Resources/config/provider.xml

```diff
@@ -63,7 +63,7 @@
             <argument/>
             <argument/>
             <argument type="service" id="sonata.media.thumbnail.format"/>
-            <argument type="service" id="sonata.media.buzz.browser"/>
+            <argument type="service" id="sonata.media.http.client"/>
```

Comment:
this could be a tricky bit to keep BC. maybe a compilar pass that detects if buzz.browser service exists (legacy system) and in that case removes sonata.media.http.client service and instead sets an alias to the buzz.browser service.